src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -215,6 +216,9 @@ applyCmd v logger gasLogger pdbenv miner gasModel txCtx spv cmd initialGas mcach
       cr <- catchesPactError $! runPayload cmd managedNamespacePolicy
       case cr of
         Left e
+          | chainweb217Pact' -> do
+            r <- jsonErrorResult (PactError ArgsError def [] "") "tx failure for request key when running cmd"
```

Comment:
I don't know how fine-grained the information in `PactError` is. Would it make sense to include more high-level some info about the failure in the message? Is there some classification of failures? Anything that doesn't require fragile or unstable formatting.